docs/index.md

```diff
@@ -1,121 +1,116 @@
-                        </svg>
-                        Ibexa DXP v4.4 is now out!
-                    </span>
-                    <span class="d-flex flex-column flex-xs-row flex-sm-row flex-md-row justify-content-between align-items-center ml-md-4">
-                        <a href="release_notes/ibexa_dxp_v4.4" class="btn btn-sm btn-announcement-outline mr-4 ml-4 ml-md-0 mb-3 mb-sm-0">
-                            See more
-                        </a>
-                    </span>
+        <div class="col-12 d-flex flex-column flex-md-row">
+            <div class="info-tile-up" id="tile2">
+                <div class = "info-tile-title"><a href="release_notes/ibexa_dxp_v4.5/" style="color: #af1164;">The latest release <span class="pill">v4.5</span></div>
+                <div class="info-tile-body align-middle">
+                        <a>The <a href="release_notes/ibexa_dxp_v4.5/" style="color: #af1164;">latest version of Ibexa DXP is v4.5</a>. You can now <a href="update_and_migration/from_4.4/update_from_4.4/" style="color: #af1164;">update your application</a> to the latest version.
                 </div>
             </div>
         </div>
     </div>
-    <div class="row">
-        <div class="col-lg px-2 px-lg-4">
-            <div class="tile">
-                <div class="row">
-                    <div class="col-lg">
-                        <h5 class="tile-title">
-                            <svg class="tile-icon align-middle" width="32" height="32">
-                                <use fill="var(--ibexa-jazzberry)" xlink:href="images/ez-icons.svg#publish"></use>
-                            </svg>
-                            Installation
-                        </h5>
-                        <div class="tile-body">
-                            <ol>
-                                <li><a href="getting_started/requirements/">Requirements</a></li>
-                                <li><a href="getting_started/install_ibexa_dxp/">Install Ibexa DXP</a></li>
-                                <li><a href="getting_started/first_steps/">First steps</a></li>
-                            </ol>
-                        </div>
-                    </div>
+</div>
+    </div> 
+        <div class="accordion">
+            <details>
+                <summary>
+                    <img class="summary-icon" src="images/Build site.svg" height="25px" width="25px">Notable changes in v4.5</img></summary>
```

Comment:
It would be good to fix indentation in the file because good indentation later helps a lot. 🙂 
```suggestion
                    <img class="summary-icon" src="images/Build site.svg" height="25px" width="25px">Notable changes in v4.5</img>
                </summary>
```